11-getting-started-with-Jupyter.Rmd

```diff
@@ -0,0 +1,350 @@
+a university, a company, *etcetera*) and require authentication to gain 
+access. <!--- Insert link to public JupyterHub here if we can get 
+access/permission (either 2i2c collaboration, or 
+https://notebooks.gesis.org/hub/home) --> Jupyter can also be installed on your
+own computer (desktop or laptop), and we provide examples of how to do this in
+chapter \@ref(move-to-your-own-machine).
+
+## Code cells
+
+The sections of a Jupyter notebook that contain code in the R programming 
+language are referred to as code cells. A code cell that has not yet been executed 
+has no number inside the square brackets to the left of the cell 
+(Figure \@ref(fig:code-cell-not-run)). Running a code cell will execute all of 
+the code it contains, and the output (if any exists) will be displayed directly
+underneath the code that generated it. Outputs may include printed text or 
+numbers, data frames and data visualizations. Cells that have been executed 
+also have a number inside the square brackets to the left of the cell. 
+This number indicates the order in which the cells were run 
+(Figure \@ref(fig:code-cell-run)).
+
+```{r code-cell-not-run, echo = FALSE, fig.cap = "A code cell in Jupyter that has not yet been executed.", fig.retina = 2, out.width="100%"}
+image_read("img/code-cell-not-run.png") %>% 
+  image_crop("3632x1000")
+```
+
+```{r code-cell-run, echo = FALSE, fig.cap = "A code cell in Jupyter that has been executed.", fig.retina = 2, out.width="100%"}
+image_read("img/code-cell-run.png") %>% 
+  image_crop("3632x2000")
+```
+
+
+### Executing code cells
+
+Code cells can be run independently or as part of executing the entire notebook using one of the "**Run all**" commands found in the **Run** or **Kernel** menus in Jupyter. Running a single code cell independently is a workflow typically used when editing or 
+writing your own R code. Executing an entire notebook is a workflow typically 
+used to ensure that your analysis runs in its entirety before sharing it with
+others, and when using a notebook as part of an automated process.
+
+To run the a code cell independently, the cell needs to first be activated. This
+is done by clicking on it with the cursor. Jupyter will indicate a cell has been
+activated by highlighting it with a blue rectangle to its left. After the cell
+has been activated, the cell can be run by either pressing the **Run** (▶) 
+button in the Jupyter notebook tab menu, or by using a keyboard shortcut of 
+`Shift + Enter`.
+
+```{r activate-and-run-button, echo = FALSE, fig.cap = "An activated cell that is ready to be be run. The red arrow points to blue rectangle to the cell's left. The blue rectangle indicates that it is ready to be run. This can be done by clicking the run button (circled in red).", fig.retina = 2, out.width="100%"}
+image_read("img/activate-and-run-button-annotated.png") %>% 
+  image_crop("3632x900")
+```
+
+To execute all of the code cells in an entire notebook, you can either select 
```

Comment:
I think for sections like this where there are a lot of steps, it would be helpful to have these commands listed as bullets or a numbered list (similar to how the kernel section below has the steps in bullets)

Reply:
I actually think anywhere we have "steps" for using Jupyter it would be helpful to have these stand out from the paragraph text (maybe in a box or as a numbered list/bullets) so someone who wants to reference how to do something they can easily find the steps in the text

Reply:
Done in e7d14da for this specific one. Will keep my eyes out for other places to add this here and in other chapters.